sat/cli/bootsys/bos.py

```diff
@@ -943,3 +953,27 @@ def do_bos_boots(args):
     except BOSFailure as err:
         LOGGER.error(err)
         sys.exit(1)
+
+
+def do_bos_reboots(args: Namespace):
+    """Reboots compute nodes and UANs using BOS.
+
+    Args:
+        args: The argparse.Namespace object containing the parsed arguments
+            passed to this stage.
+
+    Returns: None
+    """
+    if not args.disruptive:
+        prompt_continue('reboot of compute nodes and UANs using BOS')
+
+    try:
+        do_bos_operations(
+            "reboot",
+            get_config_value("bootsys.bos_boot_timeout"),
```

Comment:
I'm not exactly sure how the reboot operation works in BOS, but I would expect that the BOS reboot operation might take as long as a shutdown operation followed by a boot operation. By that logic, I think it would make sense to add the `bos_boot_timeout` and the `bos_shutdown_timeout` here. That is:

```suggestion
            get_config_value('bootsys.bos_boot_timeout') +
            get_config_value('bootsys.bos_shutdown_timeout'),
```

Alternatively, we could add a new `bos_reboot_timeout`, but I don't know if it's necessarily any better to have that separate from the shutdown and boot timeouts. FWIW, all the timeouts really do is cause the `sat bootsys` command to stop waiting and return to the user. This is arguably undesirable because I believe the BOS operation itself has its own independent timeout logic. The main thing we want to avoid here is having the `sat bootsys reboot --stage bos-operations` command time out because the timeout was shorter than BOS's timeout for a reboot operation.

Reply:
Understood, yes I did notice the timeout was much shorter. I will try with your suggestion.